api/v1alpha1/akodeploymentconfig_types.go

```diff
@@ -152,6 +226,31 @@ type AKOIngressConfig struct {
 	NodeNetworkList []NodeNetwork `json:"nodeNetworkList,omitempty"`
```

Comment:
ako 1.5: `NodeNetworkList` is in `NetworksConfig`

Reply:
Considering backward-compatibility, I'd rather to keep this way. We can discuss this.